### PR TITLE
Make split string consistent over Android 

### PR DESCRIPTION
**Pull request by jingmiao-z**
split method, delete "" first empty string.

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
@AppInventorWorkerBee ok to test

File: appinventor/components/src/com/google/appinventor/components/runtime/util/JavaStringUtils.java [hunk 1]
```diff
@@ -194,6 +195,14 @@ public static String joinStrings(List<Object> listOfStrings, String separator) {
     return join(listOfStrings, separator);
   }
 
+  public static List<String> split(String text, String at){
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/util/JavaStringUtils.java:198]: To address the broken test, we should change the return type here to YailList and then call YailList.makeList(...) to wrap the output of this function as a YailList.

**Review by ewpatton — CHANGES_REQUESTED** [review]

**Comment by ewpatton** [comment]
Still need to test on SDK versions pre-26 and the latest SDK.

**Review by ewpatton — APPROVED** [review]

**Pull request merged**